packages/language/src/document.js

```diff
@@ -745,12 +964,19 @@ Document.formatSentiment_ = function(sentiment, verbose) {
  */
 Document.formatTokens_ = function(tokens, verbose) {
   if (!verbose) {
-    tokens = tokens.map(function(token) {
-      return {
-        text: token.text.content,
-        partOfSpeech: Document.PART_OF_SPEECH[token.partOfSpeech.tag],
-        partOfSpeechTag: token.partOfSpeech.tag
-      };
+    tokens = tokens.map(function(rawToken) {
+      var token = extend({}, rawToken.partOfSpeech, {
+        text: rawToken.text.content,
+        partOfSpeech: Document.PART_OF_SPEECH[rawToken.partOfSpeech.tag]
+      });
+
+      for (var part in token) {
+        if (token.hasOwnProperty(part) && /UNKNOWN/.test(token[part])) {
+          token[part] = false;
```

Comment:
I think that makes sense, truthfully I could go either way - all other possible values are strings so it's likely that our users will treat it as a falsey value, in which case `undefined` and `false` are going to behave in the same fashion.